examples/music.py

```diff
@@ -0,0 +1,254 @@
+import math
+import re
+
+import discord
+from discord.ext import commands
+import lavalink
+from lavalink.audio_events import DefaultEventAdapter
+
+time_rx = re.compile('[0-9]+')
+
+
+class Music:
+    def __init__(self, bot):
+        self.bot = bot
+        self.adapters = {}
```

Comment:
Store this within the lavalink client/player object for convenience. Having to instantiate multiple things increase complexity and may lead to bad UX when something doesn't work as expected.